### DISCUSSION TITLE
Practice and Main Trial Help

### DISCUSSION THREAD

**Discussion by batshell** [jspsych/jsPsych#1612] [Q&A]
Suppose I want to make a practice trial (10 times with different values), and then I need to check the accuracy in those trial, and I need to go to the main block if the accuracy is above, say 70%, if not, I need to re-run the practice trial once more. Is there something I can do?

**Comment by jodeleeuw** [comment]
@becky-gilbert covered a situation like this in the third session of the [moving research online workshop](https://www.movingresearchonline.info).

You can watch the [session on YouTube](https://www.youtube.com/watch?v=LP7o0iAALik). This bit is discussed around the 45 minute mark.

**Reply by batshell** [reply]
Thank you, it helped me a lot!
